cmake/CMakeLists.txt

```diff
@@ -642,6 +643,22 @@ if(BUILD_TOOLS)
   install(FILES ${LAMMPS_DOC_DIR}/msi2lmp.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 endif()
 
+if(BUILD_LAMMPS_SHELL)
+  include(FindPkgConfig)
```

Comment:
I would use `find_package(PkgConfig)` here, more consistent  with the rest of the code.